michael-le.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<html lang ="en">
+<head>
+  <title>Of Patterns and Power: Web Standards Then & Now</title>
+  <meta name="description" content="Of Patterns and Power:Web Standards Then and Now">
+</head>
+<body>
+  <nav>
+    <ul>
+      <li><a href="https://medium.com/">"Medium"</a></li>
+      <li><a href="https://medium.com/let-me-repost-that-for-you-zeldman?source=logo-lo_VJihZgXj1Lp9---aa4f7024d394">""@zeldman"</a></li>
+      <li><a href="https://medium.com/let-me-repost-that-for-you-zeldman?source=logo-lo_VJihZgXj1Lp9---aa4f7024d394">"follow"</a></li>
+      <li><a href="https://medium.com/m/signin?redirect=https%3A%2F%2Fmedium.com%2Flet-me-repost-that-for-you-zeldman%2Fof-patterns-and-power-web-standards-then-now-afdb3474c681&source=--------------------------nav_reg&operation=login">"sign in"</a></li>
+      <li><a href="https://medium.com/m/signin?redirect=https%3A%2F%2Fmedium.com%2Flet-me-repost-that-for-you-zeldman%2Fof-patterns-and-power-web-standards-then-now-afdb3474c681&source=--------------------------nav_reg&operation=register">"get started"</a></li>
+    </ul>
+
+  </nav>
+  <section>
+	   <header>
+       <img src="patter-power.jpg">
+       <h1>Of Patterns and Power:Web Standards Then and Now-Medium</h1>
+		 </header>
+     <article>
+       <p> IN <a href="http://danielmall.com/articles/content-display-patterns/">“CONTENT Display Patterns”</a> (which all front-end folk should read), Dan Mall points to a truth not unlike the one <a href="https://24ways.org/2015/putting-my-patterns-through-their-paces/">"Ethan Marcotte shared last month on 24 ways"</a>. It is a truth as old as standards-based design: Construct your markup to properly support your content (not your design).</p>
```

Comment:
`<article>` should wrap your main content, in other words, you should only have _one_ pair of article tags.